content/docs/capabilities/getting-users-identity.md

```diff
@@ -220,6 +220,12 @@ The Pomerium JWT contains at least the following claims:
 | `groups` | The user's group memberships (if supported for the identity provider). |
 | `name` | The user's full name, as specified by the identity provider. |
 
+:::tip Prevent session replay attempts
+
+The `jti` claim (the JWT ID) contains a unique identifier assigned to each Pomerium JWT. You can prevent session replay attempts by implementing a system that monitors `jti` reuse either in real time or by checking session logs.
```

Comment:
One other comment — the change to unique `jti` values has been merged but not yet released, so I think this should not be backported to the v0.26 docs site.

We should probably also correct the information in the `jti` row of the table above — in v0.26 and earlier this contains the Pomerium session ID.

Reply:
OK. I'll keep this in mind going forward. I can add the previous session ID description and keep these changes in `main`.